### ISSUE TITLE
Servbot sprites are disproportionately small compared to other borg sprites, giving them a disproportionately small "hitbox"

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#11394]
![dreamseeker_2016-08-11_22-30-58](https://cloud.githubusercontent.com/assets/6526157/17610362/b36bea54-6013-11e6-88f3-037093b2488f.png)
Yes I'm actually complaining about this
I have seen people pick this sprite for the explicit purposes of powergaming via a smaller sprite, which makes them slightly (but not negligibly) harder to flash/laser/whatever.
I don't think I've seen any module except Secborg pick them too, so there's definitely a correlation

**Comment by Exxion** [comment]
I call bullshit on that making them harder to laser
But it would affect any kind of melee

**Comment by Kurfursten** [comment]
Yeah, no body actually clicks their target when lasering, they click the floor

But melee flashing is true

Borgs get "slipped" and shoot sparks when they pass over a water tile when?

**Comment by Exxion** [comment]
I'm not really sure what a good way to fix this would be
We could give the sprite a transparent area about the size of a regular borg, but that would cause problems with trying to interact with things under the borg.

**Comment by NigglyWiggly** [comment]
I wanted to do that when I made them, but I wasn't really sure how

**Comment by Megamaw** [comment]
i don't think this is a major issue, personally, but let's see what people think

**Comment by 9600bauds** [comment]
It's a small enough of an issue with no obvious or simple solution that probably nothing will ever be done about it but it'd be nice to have a guideline of sorts for future borg sprites
It's undeniable that people have and do powergame with this though

**Comment by ghost** [comment]
You can have an invisible mouse opacity = 1 overlay for all borgs of the same size acting as a standardising hitbox of sorts I think would work
